packages/markers/src/browser/problem/problem-manager.spec.ts

```diff
@@ -45,6 +46,7 @@ before(() => {
     testContainer.bind(ProblemManager).toSelf();
 
     manager = testContainer.get(ProblemManager);
+    manager.cleanAllMarkers();
```

Comment:
That line doesn't seem to help, you still need to call this in the tests...

Reply:
@marechal-p I wanted to add the `clean` before we set the markers with `setMarkers` (a little lower) to ensure that we have a clean state on each test run. For some of the tests you are correct that I perform a `clean` but this is to test the behavior when no markers are present by clearing those present in the `beforeEach`.

Reply:
Ok I understand better now! So by default it adds 2 markers, but in some tests you want none.

Reply:
I still think that this line is not useful in the `beforeEach` hook because we create a new manager everytime.